src/Tasks.UnitTests/GetComAssembliesMetadata_Tests.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+#if FEATURE_APPDOMAIN
```

Comment:
```suggestion
#if NETFRAMEWORK
```

As the new task is not really related to appdomains.